contrib/openbsm/bin/bsmconv/linau.h

```diff
@@ -0,0 +1,120 @@
+#ifndef _LINAU_H_
+#define _LINAU_H_
+
+#include <sys/nv.h>
+#include <sys/queue.h>
+
+#include <stdint.h>
+#include <stdio.h>
+
+
+struct linau_event {
```

Comment:
We should hide these structures' definitions in linau_impl.h and provide an API to access its elements if it's needed.